Update endpoints

takayukioda opened the pull request:
I wanted to have same format with other api challenges, and form of
`/api/***/***` is more common as REST api, so I updated.
Addition to that, separating endpoint and it's method make hard to
understand the api, so I remove endpoint setting in config file

@NckHmml please review me.
I will update js server later on.

takayukioda committed:
Update endpoints

I wanted to have same format with other api challenges, and form of
`/api/***/***` is more common as REST api, so I updated.
Addition to that, separating endpoint and it's method make hard to
understand the api, so I remove endpoint setting in config file

NckHmml commented:
I originally did this so it would be easier for the user to use his own paths.
And, IMO, if we end up following the RESTful specifications we should also combine register and delete to a single path

Anyhow,
LGTM

takayukioda committed:
Update method

Accessing endpoint `/api/accounts/disable` with DELETE method doesn't
make sense, so.
